fm.reaper.Reaper.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Reaper
+Exec=/app/REAPER/reaper
```

Comment:
you instead have a script or a symlink to this in `/app/bin` and just put the name of that. Not an absolute path to a non standard location.

Reply:
When I put only the name in it I get the error: `WARNING: Binary not found for Exec line in /var/home/cleomenezesjr/Projects/Reaper/build/export/share/applications/fm.reaper.Reaper.desktop: reaper`

Reply:
as I said, have a symlink in `/app/bin`. Or wrapper script.